src/e2e-test/features/dataplex/sink/DesignTimeDataplex.feature

```diff
@@ -0,0 +1,91 @@
+    And Click plugin property: "dataplexUpdateTableSchema"
+    And Click plugin property: "dataplexRequirePartitionFilter"
+    And Validate "Dataplex" plugin properties
+
+  @BATCH-TS-DATAPLEX-DSGN-02
+  Scenario: To verify user should be able to validate fields with operation radio button, update table schema
+  toggle button and integer partitionin in dataplex sink with when asset type is selected as BigQuery dataset
+    When Open Datafusion Project to configure pipeline
+    And Select Sink plugin: "Dataplex" from the plugins list
+    And Navigate to the properties page of plugin: "Dataplex"
+    And Enter input plugin property: "dataplexReferenceName" with value: "dataplex.referenceName"
+    And Replace input plugin property: "dataplexProjectID" with value: "dataplex.project"
+    And Enter input plugin property: "dataplexLocationID" with value: "dataplex.location"
+    And Enter input plugin property: "dataplexLakeID" with value: "dataplex.lake"
+    And Enter input plugin property: "dataplexZoneID" with value: "dataplex.zone"
+    And Enter input plugin property: "dataplexAssetID" with value: "dataplex.assetBQ"
+    And Enter input plugin property: "dataplexTableName" with value: "dataplex.outputTable"
+    And Select radio button plugin property: "dataplexPartitioningType" with value: "dataplex.integerPartitioningRadioBtn"
+    And Click plugin property: "dataplexUpdateTableSchema"
+    And Validate "Dataplex" plugin properties
+
+  @BATCH-TS-DATAPLEX-DSGN-03
+  Scenario Outline: To verify user should be able to validate mandatory fields in dataplex sink with Storage bucket
+  when different formats are selected like avro csv json
+    When Open Datafusion Project to configure pipeline
+    And Select Sink plugin: "Dataplex" from the plugins list
+    And Navigate to the properties page of plugin: "Dataplex"
+    And Enter input plugin property: "dataplexReferenceName" with value: "dataplex.referenceName"
+    And Replace input plugin property: "dataplexProjectID" with value: "dataplex.project"
+    And Enter input plugin property: "dataplexLocationID" with value: "dataplex.location"
+    And Enter input plugin property: "dataplexLakeID" with value: "dataplex.lake"
+    And Enter input plugin property: "dataplexZoneID" with value: "dataplex.zone"
+    And Enter input plugin property: "dataplexAssetID" with value: "dataplex.assetStorageBucket"
+    And Select radio button plugin property: "dataplexAssetType" with value: "dataplex.storageBucketRadioBtn"
+    And Enter input plugin property: "dataplexTableName" with value: "dataplex.outputTable"
+    And Select dropdown plugin property: "dataplexFormat" with option value: "<Format>"
+    And Validate "Dataplex" plugin properties
+    Examples:
+      | Format      |
+      | avro        |
+      | csv         |
+      | json        |
+
+  @BATCH-TS-DATAPLEX-DSGN-06
+  Scenario Outline: Verify user should be able to validate mandatory fields with storage bucket when orc and parquet
```

Comment:
Why are we testing orc and parquet separately? Why not just add them in the previous scenario?

Reply:
To test orc and parquet in Dataplex sink plugin, input schema is mandatory. So, there are additional steps in this scenario to add input schema from source.